samples/arm-none-eabi/reported.by

```diff
@@ -0,0 +1,310 @@
+reporter_name="Jasmin Jessich"
+reporter_url="http://jasmin.anw.at/"
+reporter_comment="NOTE: Please read samples/arm-none-eabi/reported.by prior to any built!"
+
+#
+# This configuration will build an arm-none-eabi compiler to be used for
+# embedded development. The original build script can be found here:
+#    https://launchpad.net/gcc-arm-embedded
+# To see the original build script, you have to download the installer:
+#    https://launchpad.net/gcc-arm-embedded/5.0/5-2015-q4-major
+# 
+# I tried to mimic the original configure options and environment settings with
+# crosstool-ng (CT_NG) as close as possible. I really analysed every option
+# which was not provided by CT-NG or which was added. The description in this
+# file explains each diversion from the original.
+# 
+# NOTE:   newlib-nano is currently not built!
+# 
+# The original uses the git/SVN versions of binutils-gdb, gcc and newlib. CT-NG
+# doesn't currently support using a git or SVN repository for the source
```

Comment:
> Can you check if you have something in your environment that you forgot to add to this
> branch?

Your guess was really good ;)

> Perhaps, add this new sample to .travis.yml and see if Travis CI can build it.

I didn't this on purpose. The sample is new and Travis CI runs very long with the current samples already. But I agree with you, then I would have seen my mistake.
Let's wait until it is merged to Trunk with that.